.github/workflows/fetcher.yml

```diff
@@ -7,27 +7,33 @@ on:
   workflow_dispatch:
 
 jobs:
+  gemfilesgetter: 
+    uses: stefanomunarini/relaton-data-rfcs/.github/workflows/gemfiles-getter.yml@main
   fetch:
+    needs: [gemfilesgetter]
     name: rfcsubseries rfc fetching
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Get BibXML service’s Relaton version
-        run: echo "RELATON_VERSION=$(curl -L https://github.com/ietf-ribose/bibxml-service/raw/main/RELATON_VERSION.txt)" >> $GITHUB_ENV
-      - name: Test BibXML service’s Relaton version
-        run: echo $RELATON_VERSION
+        uses: actions/checkout@v3
       - name: Use Ruby
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 3.0
-      - name: Install gem
-        run: |
-          gem install relaton-cli  -v "~> $RELATON_VERSION"
+      - name: ls
+        run: ls
+      - name: bundle add relaton-cli
+        run: bundle add relaton-cli --github 'ietf-ribose/bibxml-service' --ref 'gemfilelock'
+      - name: Install Gems
+        run: bundle install
+      - name: Verify Gems versions
+        run: bundle list
+      - name: Test relaton Gem
+        run: bundle info relaton
       - name: Fetch documents
         run: |
           rm -rf data
-          relaton fetch-data ietf-rfc-entries
+          bundle exec relaton fetch-data ietf-rfc-entries
```

Comment:
@CAMOBAP the biggest issue is here. The workflow fails here with error 

> "bundler: command not found: relaton
> [6](https://github.com/stefanomunarini/relaton-data-rfcs/actions/runs/4228652442/jobs/7344294032#step:9:7)
> Install missing gem executables with `bundle install`". 

The step before (bundle info relaton), however, returns the correct info for the relaton gem.

Reply:
This happens because 'relaton-cli` "comes" from the current repo https://github.com/stefanomunarini/relaton-data-rfcs/actions/runs/4229421757/jobs/7345783595#step:6:149

But I see more global problems with the approach, which is probably because of misunderstanding the ruby toolchain:
1. There is no reason to duplicate `relaton-cli` gem in other sources than https://github.com/relaton/relaton-cli
2. Like any other programming language ruby have the tooling to manage/install dependencies. In the python world we use `pip` which read info from `requirements.txt`, in the ruby world we use `bundler` which read info from `Gemfile`/`Gemfile.lock`
3. 

My strong proposal:
- Find a place for `Gemfile` / `Gemfile.lock` in any repo which you want. `Gemfile` should not contain `gemspec` line in it and probably should look like this:
  ```ruby
  # frozen_string_literal: true
  source "https://rubygems.org"

  gem 'relaton-cli', "=1.2.3" # you can specify exact version that you need or pick github ref
  ```
-  Github action workflow in `relaton-data-*` will download that `Gemfile`/`Gemfile.lock`
   - run `bundle install`
   - run `bundle exec relaton ...`
   - commit files in data repository

@stefanomunarini feel free to ping me here or in Skype if you have some questions